java/README.md

```diff
@@ -59,6 +59,19 @@ For any other library, such as OkHttp, you will need to include the correspondin
 from the [instrumentation project](https://github.com/open-telemetry/opentelemetry-java-instrumentation) and
 modify your code to initialize it in your function.
 
+### Configuring Context Propagators
+
+#### If you emit your traces to AWS X-Ray (instead of a third-party service) and have enabled X-Ray Active Tracing
+Please use the following environment variable for your lambda:
+`OTEL_PROPAGATORS=tracecontext,baggage,xray-lambda`
+
+#### If you emit your traces to another system besides AWS X-Ray (Default Setting)
```

Comment:
Should I instead mention "If you emit your traces to another system besides AWS X-Ray or have disabled X-Ray Active Tracing"

Reply:
I don't think the extra distinction is of consequence as `xray-lambda` will function the same as `xray` if Active Tracing is disabled.

Reply:
Got it, will leave as is then